convert/plugin_updates.go

```diff
@@ -94,3 +99,90 @@ func autoGenerateNamespaceForRLAPluginConsumerGroups(plugin *kong.ConsumerGroupP
 	}
 	return nil
 }
+
+func updatePlugins(content *file.Content) {
+	for idx := range content.Plugins {
+		plugin := &content.Plugins[idx]
+		updateLegacyPluginConfig(plugin)
+	}
+
+	for _, service := range content.Services {
+		for _, plugin := range service.Plugins {
+			updateLegacyPluginConfig(plugin)
+		}
+	}
+
+	for _, route := range content.Routes {
+		for _, plugin := range route.Plugins {
+			updateLegacyPluginConfig(plugin)
+		}
+	}
+
+	for _, consumer := range content.Consumers {
+		for _, plugin := range consumer.Plugins {
+			updateLegacyPluginConfig(plugin)
+		}
+	}
+
+	for _, consumerGroup := range content.ConsumerGroups {
+		for _, plugin := range consumerGroup.Plugins {
+			updateLegacyPluginConfig(&file.FPlugin{
+				Plugin: kong.Plugin{
+					ID:     plugin.ID,
+					Name:   plugin.Name,
+					Config: plugin.Config,
+				},
+			})
+		}
+	}
+}
+
+func updateLegacyPluginConfig(plugin *file.FPlugin) {
+	if plugin != nil && plugin.Config != nil {
+		config := plugin.Config.DeepCopy()
+
+		config = updateLegacyFieldToNewField(config, "blacklist", "deny", "")
```

Comment:
We don't want to print plugin name to console here?

Reply:
Many plugins require these updates. So, I left it out.
I have added changes so that pluginName gets printed now.